openapi.go

```diff
@@ -24,15 +24,15 @@ func NewOpenAPI() *OpenAPI {
 	return &OpenAPI{
 		description:            &desc,
 		generator:              openapi3gen.NewGenerator(),
-		globalOpenAPIResponses: []openAPIError{},
+		globalOpenAPIResponses: []openAPIResponse{},
 	}
 }
 
 // Holds the OpenAPI OpenAPIDescription (OAD) and OpenAPI capabilities.
 type OpenAPI struct {
 	description            *openapi3.T
 	generator              *openapi3gen.Generator
-	globalOpenAPIResponses []openAPIError
+	globalOpenAPIResponses []openAPIResponse
 }
 
```

Comment:
No need for it now. We can send it as a global `routeOption` that will be set at the Route creation time!

But maybe for another PR.

Reply:
Assuming you agree with https://github.com/go-fuego/fuego/pull/276#discussion_r1887167481

This will just be removed after deprecation after the deprecation notice of this thing on the next release.